pages/links/index.js

```diff
@@ -0,0 +1,120 @@
+import { NextSeo } from "next-seo"
+
+function LinksPage() {
+  return (
+    <>
+      <NextSeo
+        title="Tech description of portfolio"
+        description="This portfolio was fully designed and built by Halyna Yavtushenko. It consists of 3 repositories: website, admin, and backend. For the website, used Next.js framework. For admin - React library, Redux + RTK Query, Tailwind. For server-side - Express.js."
+      />
+      <div className="relative min-h-screen bg-gray-900 text-white flex flex-col items-center justify-center p-4">
+        {/* Conditional Background Images */}
+        <picture className="absolute inset-0 w-full h-full z-0 filter blur-md">
+          <source
+            media="(min-width: 1024px)"
```

Comment:
Load background based on media-query. Looks like image from nextjs isn't recommended for background since doesn't have this ability to load by media-query